start-site/src/main/resources/application.yml

```diff
@@ -199,12 +207,26 @@ initializr:
       spring-releases:
         name: Spring Releases
         url: https://repo.spring.io/release
+      spring-milestones:
+        name: Spring Milestones
+        url: https://repo.spring.io/milestone
       sonatype-snapshots:
         name: Sonatype Snapshots
         url: https://oss.sonatype.org/content/repositories/snapshots/
         releasesEnabled: false
         snapshotsEnabled: true
   dependencies:
+    - name: Architecture
+      content:
+        - name: Spring Modulith
```

Comment:
We don't name things with "Spring" in front of it. Look at other entries for more details.

Reply:
Are you sure about that? I see all Spring Data flavors using the prefix, Spring Integration, Spring Batch etc.

Reply:
My mistake, this was my recollection before we did some harmonization. Please disregard this.

Reply:
All good, no worries.